integrations/symfony-full-configuration.rst

```diff
@@ -65,12 +65,24 @@ This page shows an example of all configuration values provided by the bundle.
             stopwatch:
                 enabled: true
                 stopwatch: 'debug.stopwatch'
+
+        toolbar:
+            enabled: true
```

Comment:
> this should be kernel.debug, no?

No. kernel.debug=false will turn off the toolbar. 
kernel.debug=true allows you to control the toolbar with toolbar.enabled

>  or is it only ever asked for if the debug toolbar itself is enabled?

correct